backend/src/handlers.rs

```diff
@@ -0,0 +1,41 @@
+use crate::gamestate::game_repo::{GameRepo, StableGameRepo};
+use actix_web::{post, web, HttpResponse, Responder};
+use serde::Deserialize;
+use serde::Serialize;
+use std::sync::{Arc, Mutex};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct GamePostData {
+    name: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Response {
+    gameID: String,
+    server: String,
+    token: String,
+}
+
+#[post("/game")]
+pub async fn create_game(
+    data: web::Data<Arc<Mutex<StableGameRepo>>>,
+    body: web::Json<GamePostData>,
+) -> impl Responder {
+    let result = data.lock().unwrap().create_game(body.name.clone()).await;
```

Comment:
`result` je příliš generické jméno, na první pohled by mělo být jasné, co to je.